community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -691,7 +691,13 @@ object projects:
 
   lazy val fs2 = SbtCommunityProject(
     project = "fs2",
-    sbtTestCommand = "coreJVM/test; coreJS/test",  // io/test requires JDK9+
+    sbtTestCommand = List(
```

Comment:
This seems like a quick workaround but maybe we could try to do a little better and attempt to bump fs2 version first and see if this got fixed on their side? Our managed community build seems to be using a version from two years ago.
And it looks like in the newest version of fs2 was moved from `jvm` and `js` to `shared`. So if we only patch it here, this will have no effect if we bump fs2 version later at some point in the future

Reply:
Good idea, but I'm afraid I don't have write access to `dotty-staging/fs2` so cannot sync it with the upstream repo. @hamzaremmal would you be able to do it and update the submodule in this project? 

Overall the issue seems to be rare (we know only about a single failure), so we might delay suppressing failing test for some more time now.

Reply:
fyi @mpilquist

Reply:
@WojciechMazur I will update `dotty-staging/fs2` over the weekend